tracer/src/Datadog.AzureFunctions/README.md

```diff
@@ -0,0 +1,12 @@
+# Datadog Instrumentation Library for .NET Azure Functions
+## `Datadog.AzureFunctions`
+
+**Note: This is an unsupported pre-release package.**
+
+**Note: If your Azure Functions are running on an App Service plan (aka Dedicated plan) on Windows, use the [Datadog Azure App Services Site Extension](https://docs.datadoghq.com/serverless/azure_app_services/azure_app_services_windows/?tab=net) instead.**
+
+Add this package to your Azure Functions project to enable Datadog APM tracing.
```

Comment:
I think we should be very explicit about exactly which scenarios this nuget is for, and which it's _not_ for.

e.g. 
- App Service plan on Windows
- App service plan on Linux
- App service plan on Linux container
- Consumption plan
etc etc

ideally a markdown table with cross/tick or something😄

Reply:
I started a (WIP) table.